ivy/functional/frontends/numpy/statistics/order_statistics.py

```diff
@@ -104,3 +108,29 @@ def nanpercentile(
                     arrayofpercentiles.append(_cpercentile(ii, i))
                 resultarray.append(arrayofpercentiles)
         return resultarray
+
+
+
+@ivy.numpy_frontend

```

Comment:
I'm not sure what the purpose of this decorator is. Please could you point me to the documentation regarding this as I tried searching in the deep dive docs and couldn't find it

Reply:
it will first check to see if the input arrays are NumPy arrays. But I removed it as I think the function itself is checking and converting it.